docs/v2/10_webgl-browser.md

```diff
@@ -75,7 +75,7 @@ public class WebGLGetGasPrice : MonoBehaviour
 
 ### Gas Limit {#gas-limit}
 
-Get the current gas limit for a transaction based on chain / network and RPC.
+Get the **current gas limit** for a transaction based on chain / network and RPC. Used in combination with the gas price of Ethereum to estimate the transaction fees required to execute certain actions on the Ethereum blockchain within the game, such as transferring assets between players or minting new in-game NFTs. For more on gas limits, see [What is gas limit?](https://ethereum.org/en/developers/docs/gas/#what-is-gas-limit)
```

Comment:
This description was not updated

Reply:
Gas limit is defined by the maximum amount of gas a user can consume to conduct a transaction

Reply:
I'm kinda confused becuz in my latest commits shown here (https://github.com/ChainSafe/game-docs-v2/pull/37/commits) it says I did. But the convo is still revolving around an older commit?

Reply:
merging to see what happens